### ISSUE TITLE
Binaries with inserted commands do not round-trip

### ISSUE THREAD

**Issue by jonpalmisc** [cxnder/ktool#15]
Attempting to insert a load command into a binary produced by `ktool` after previously inserting a load command does not work. 

To reproduce, run the following:
```sh
$ ktool insert --lc load --payload libFirst.dylib --out FirstOutput Target
$ ktool insert --lc load --payload libSecond.dylib --out SecondOutput FirstOutput
```

This results in the following error:

```
Traceback (most recent call last):
  File "/opt/homebrew/bin/ktool", line 866, in <module>
    main()
  File "/opt/homebrew/bin/ktool", line 356, in main
    args.func(args)
  File "/opt/homebrew/bin/ktool", line 567, in insert
    image = process_patches(image)
  File "/opt/homebrew/bin/ktool", line 407, in process_patches
    return ktool.reload_image(image)
  File "/opt/homebrew/lib/python3.9/site-packages/ktool/ktool.py", line 55, in reload_image
    return load_image(image.slice)
  File "/opt/homebrew/lib/python3.9/site-packages/ktool/ktool.py", line 84, in load_image
    return Dyld.load(macho_slice, load_symtab=load_symtab, load_imports=load_imports, load_exports=load_exports)
  File "/opt/homebrew/lib/python3.9/site-packages/ktool/dyld.py", line 289, in load
    image = Image(macho_slice)
  File "/opt/homebrew/lib/python3.9/site-packages/ktool/dyld.py", line 138, in __init__
    self.macho_header: ImageHeader = ImageHeader.from_image(macho_slice=macho_slice)
  File "/opt/homebrew/lib/python3.9/site-packages/ktool/dyld.py", line 77, in from_image
    raise ex
  File "/opt/homebrew/lib/python3.9/site-packages/ktool/dyld.py", line 73, in from_image
    load_cmd = Struct.create_with_bytes(LOAD_COMMAND_MAP[LOAD_COMMAND(cmd)], cmd_raw)
  File "/opt/homebrew/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/enum.py", line 384, in __call__
    return cls.__new__(cls, value)
  File "/opt/homebrew/Cellar/python@3.9/3.9.10/Frameworks/Python.framework/Versions/3.9/lib/python3.9/enum.py", line 702, in __new__
    raise ve_exc
ValueError: 791555631 is not a valid LOAD_COMMAND
```

Some testing on my end reveals this doesn't seem to depend on the binaries themselves.

**Comment by cxnder** [comment]
Fixed in 963e41c

**Comment by cxnder** [comment]
![image](https://user-images.githubusercontent.com/55725881/167370580-5b6492cd-8204-47f4-867d-daaa198a9d97.png)

**Comment by cxnder** [comment]
Fix published in 1.3.0
